### PR TITLE
Update policy-csp-networklistmanager.md

### PR DESCRIPTION
**Pull request by msarcletti**
Additional information on how to use and configure AllowedTlsAuthenticationEndpoints and ConfiguredTLSAuthenticationNetworkName

### CODE REVIEW

File: windows/client-management/mdm/policy-csp-networklistmanager.md [hunk 1]
```diff
@@ -58,7 +58,16 @@ manager: dansimp
 
 <!--/Scope-->
 <!--Description-->
-This policy setting provides the list of URLs (separated by Unicode character 0xF000) to endpoints accessible only within an enterprise's network. If any of the URLs can be resolved over HTTPS, the network would be considered authenticated.
+This policy setting provides the list of URLs (separated by Unicode character 0xF000) to endpoints accessible only within an enterprise's network. If any of the URLs can be resolved over HTTPS, the network would be considered authenticated.  
+When entering a list of TLS Endpoints in MEM (Microsoft Endpoint Management), you must follow this format even in the UI:
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-networklistmanager.md:62]:
```suggestion

When entering a list of TLS endpoints in Microsoft Endpoint Manager, you should follow this format, even in the UI:  
```

File: windows/client-management/mdm/policy-csp-networklistmanager.md [hunk 2]
```diff
@@ -58,7 +58,16 @@ manager: dansimp
 
 <!--/Scope-->
 <!--Description-->
-This policy setting provides the list of URLs (separated by Unicode character 0xF000) to endpoints accessible only within an enterprise's network. If any of the URLs can be resolved over HTTPS, the network would be considered authenticated.
+This policy setting provides the list of URLs (separated by Unicode character 0xF000) to endpoints accessible only within an enterprise's network. If any of the URLs can be resolved over HTTPS, the network would be considered authenticated.  
+When entering a list of TLS Endpoints in MEM (Microsoft Endpoint Management), you must follow this format even in the UI:  
+```<![CDATA[https://nls.corp.contoso.com&#xF000;https://nls.corp.fabricam.com]]>```  
+- The HTTPS endpoint must not have any additional authentication checks such as login or multi-factor authentication.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-networklistmanager.md:64]:
```suggestion
- The HTTPS endpoint must not have any additional authentication checks, such as login or multifactor authentication.
```

File: windows/client-management/mdm/policy-csp-networklistmanager.md [hunk 3]
```diff
@@ -58,7 +58,16 @@ manager: dansimp
...
+- The HTTPS endpoint must be an internal address not accessible from outside the corporate network.
+- The client must trust the server certificate, so the CA cert the HTTPS server cert chains to must be present in the client machines root certificate store.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-networklistmanager.md:66]:
```suggestion
- The client must trust the server certificate, so the CA certificate that the HTTPS server certificate chains to must be present in the client machine's root certificate store.
```

File: windows/client-management/mdm/policy-csp-networklistmanager.md [hunk 4]
```diff
@@ -58,7 +58,16 @@ manager: dansimp
...
+- A certificate should not be a public certificate.
+
+
+
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-networklistmanager.md:70]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msarcletti, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/client-management/mdm/policy-csp-networklistmanager.md [hunk 5]
```diff
@@ -58,7 +58,15 @@ manager: dansimp
 
 <!--/Scope-->
 <!--Description-->
-This policy setting provides the list of URLs (separated by Unicode character 0xF000) to endpoints accessible only within an enterprise's network. If any of the URLs can be resolved over HTTPS, the network would be considered authenticated.
+This policy setting provides the list of URLs (separated by Unicode character 0xF000) to endpoints accessible only within an enterprise's network. If any of the URLs can be resolved over HTTPS, the network would be considered authenticated.  
+
+When entering a list of TLS endpoints in Microsoft Endpoint Manager, you should follow this format, even in the UI:
```
Comment (msarcletti) [inline windows/client-management/mdm/policy-csp-networklistmanager.md:63]:
```suggestion
When entering a list of TLS endpoints in Microsoft Endpoint Manager, you must follow this format, even in the UI:  
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by denisebmsft** [comment]
Thank you @msarcletti @JohanFreelancer9

**Pull request merged**